src/bundle/Resources/views/themes/admin/user/invitation/modal.html.twig

```diff
@@ -1,5 +1,15 @@
 {% trans_default_domain 'user_invitation' %}
 
+{% set content %}
+    {% block content %}
```

Comment:
We can change `content` to something more specific, like `top_input_content` or `search_input_content`
```suggestion
{% set search_input_content %}
    {% block search_input_content %}
```